hail/python/hailtop/pipeline/docs/getting_started.rst

```diff
@@ -0,0 +1,5 @@
+.. _sec-getting_started:
+
+===============
+Getting Started
+===============
```

Comment:
the plan is to fill out all these empty sections in follow up PRs?

Reply:
Already did in this PR now.